<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>477</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="twClient">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Client Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Client Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Billing Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edtBillName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Address 1:     </string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Postal Code:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edtAddress2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Address 2 :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtPostNo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Postal #:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtAddress1"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="edtTown"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Town:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="edtEmail"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="edtPostCode"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Telephone:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edtTele"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Mobile #:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="edtMobile"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,0">
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbSalute"/>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="edtContactName"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Contact Name:     </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Contact Email:      </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtContactEmail"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Contact Mobile:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edtContactMobile"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Agency:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="cbClientGroup"/>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="cbAgency"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Client Group:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Account Manager:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="cbAccountManager"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Charges</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>561</width>
         <height>259</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0">
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="sbAgencyComm"/>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="sbSaleRepComm"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Interest Rate:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cbBillCycle"/>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="sbIntRate"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Revenue Type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Agency Commission:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Sales Rep Commission:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="sbGracePeriod"/>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="sbLateFee"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Billing Cycle:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Discount:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="sbDiscount"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbRevenueType"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Late Fee:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Grace Period:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>edtBillName</tabstop>
  <tabstop>edtAddress1</tabstop>
  <tabstop>edtAddress2</tabstop>
  <tabstop>edtPostNo</tabstop>
  <tabstop>edtTele</tabstop>
  <tabstop>edtTown</tabstop>
  <tabstop>edtEmail</tabstop>
  <tabstop>edtPostCode</tabstop>
  <tabstop>edtMobile</tabstop>
  <tabstop>cbSalute</tabstop>
  <tabstop>edtContactName</tabstop>
  <tabstop>edtContactEmail</tabstop>
  <tabstop>edtContactMobile</tabstop>
  <tabstop>cbAgency</tabstop>
  <tabstop>cbClientGroup</tabstop>
  <tabstop>cbAccountManager</tabstop>
  <tabstop>cbRevenueType</tabstop>
  <tabstop>sbDiscount</tabstop>
  <tabstop>sbAgencyComm</tabstop>
  <tabstop>sbSaleRepComm</tabstop>
  <tabstop>sbIntRate</tabstop>
  <tabstop>sbLateFee</tabstop>
  <tabstop>sbGracePeriod</tabstop>
  <tabstop>cbBillCycle</tabstop>
  <tabstop>twClient</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
